dependencies/dependencies.xml

```diff
@@ -87,6 +87,24 @@
     </releasefile>
   </dependency>
 
+  <dependency name="asio">
+    <releasefile sha1="bac61183b633a6d91ef7ae31836434bdf4f2b0d0">
+      <url>https://sourceforge.net/projects/asio/files/asio/1.12.1 (Stable)/asio-1.12.1.zip</url>
```

Comment:
Are the exact versions listed here required? Otherwise it would be "better"  to install them from the package repository on Linux systems. But i guess having them as a fall-back in-case system is to old might make sense.

More use full would be the downloads (and build scripts) for Windows

Reply:
I found out that Asio does not need to be compiled, it can be used as header-only. This makes things much easier.